awx/main/models/jobs.py

```diff
@@ -234,6 +225,7 @@ class JobTemplate(UnifiedJobTemplate, JobOptions, SurveyJobTemplateMixin, Resour
     playbook) to an inventory source with a given credential.
     '''
     SOFT_UNIQUE_TOGETHER = [('polymorphic_ctype', 'name')]
+    PASSWORD_FIELDS = ('credential', 'vault_credential')
```

Comment:
This was necessary to provide backwards compat. support for people who tried to filter/query on sensitive field values (so that it will continue to yell "can't do that!" at that them)